examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/clientlibs/cif-clientlib-base/.content.xml

```diff
@@ -3,4 +3,4 @@
     jcr:primaryType="cq:ClientLibraryFolder"
     allowProxy="{Boolean}true"
     categories="[cif-examples.base]"
-    embed="[venia-theme,cif-examples.react,core.cif.components.common,core.cif.components.product.v1,core.cif.components.productteaser.v1,core.cif.components.productcarousel.v1,core.cif.components.productcollection.v1,core.cif.components.searchbar.v1,core.cif.components.featuredcategorylist.v1.editor,core.cif.components.teaser.v1.editor,core.cif.components.navigation.v1,core.cif.components.carousel.v1]"/>
```

Comment:
Not really related to your fix but should `venia-theme` be here? Don't think so.

Reply:
Not sure, other components may need it.

Reply:
Yes we maybe should rename it. At minimum it is confusing to have something called `venia-theme`

Reply:
imho we should keep it like that, it is called venia theme because it is the venia style copied over to the examples. https://github.com/adobe/aem-core-cif-components/blob/CIF-2242/examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/clientlibs/venia-theme/.content.xml#L5